### DISCUSSION TITLE
The end of a sentence is missing.

### DISCUSSION THREAD

**Discussion by AlexBlack772** [coqui-ai/TTS#1858] [General Q&A] [answered]
I noticed that in some cases the end of a sentence is missing when voicing.

For example, when voicing the text "Hello, this is a Gradio demo for coqui-ai-TTS: a deep learning toolkit for Text-to-Speech applications and more. I can hear only a part of the sentence.

Coqui-TTS doesn't say the words "to-Speech applications and more."

My own instance of coqui tts and https://huggingface.co/spaces/akhaliq/coqui-ai-tts work similarly 

Is it known issue?

**Comment by thorstenMueller** [comment, marked as answer]
Might be `max_decoder_issue`. Can you see/adjust this value in config. Increasing the value might help.

**Reply by AlexBlack772** [reply]
thank you. it was a max_decoder_steps issue 

https://github.com/mozilla/TTS/issues/734
https://github.com/coqui-ai/TTS/commit/8c645080acc54c86cb3e9c778b71e335648c88e5
